docfx_yaml/extension.py

```diff
@@ -1446,6 +1465,100 @@ def convert_module_to_package_if_needed(obj):
                     obj['source']['remote']['repo'] == 'https://apidrop.visualstudio.com/Content%20CI/_git/ReferenceAutomation'):
                         del(obj['source'])
 
+
+                # Extract any missing cross references where applicable.
+                # Potential targets are instances of full uid shown, or
+                # if we find a short form of the uid of one of current
+                # package's items. For example:
+                # cross reference candidates:
+                #   google.cloud.bigquery_storage_v1.types.storage.SplitReadStreamResponse
+                #   SplitReadStreamResponse
+                # non-candidates:
+                #   (not from the same library)
+                #   google.cloud.aiplatform.AutoMLForecastingTrainingJob
+
+                current_name = obj["fullName"]
+                # Currently we only need to look in summary, syntax and
+                # attributes for cross references.
+                if obj.get("summary"):
+                    entry = "summary"
+                    obj[entry] = find_cross_references(obj[entry], current_name, app.env.docfx_uid_names)
+
+                if obj.get("syntax"):
```

Comment:
There is a lot of logic here. Would you mind moving all of these to another function (summary, syntax, and attributes)?

Reply:
Organized it into `search_cross_references` and renamed existing function to `convert_cross_references`.